examples/test/utils.py

```diff
@@ -92,6 +95,25 @@ def set_aml_system(olive_config):
     }
 
 
+def set_gpu_aml_system(olive_config):
```

Comment:
Is it possible to merge it with set_aml_system with new argument? Like set_aml_system(olive_config, is_gpu=True)

Reply:
Changed